pkg/util/validation/limits.go

```diff
@@ -69,6 +69,9 @@ type Limits struct {
 	MaxLocalMetadataPerMetric           int `yaml:"max_metadata_per_metric" json:"max_metadata_per_metric"`
 	MaxGlobalMetricsWithMetadataPerUser int `yaml:"max_global_metadata_per_user" json:"max_global_metadata_per_user"`
 	MaxGlobalMetadataPerMetric          int `yaml:"max_global_metadata_per_metric" json:"max_global_metadata_per_metric"`
+	// Exemplars
+	MaxExemplarsPerUser       int `yaml:"max_exemplars_per_user" json:"max_exemplars_per_user"`
```

Comment:
Two different comments here:
1. Do we want the local limit? The reason why we have a local limit for series is because of legacy reasons, but we're currently using only the global limit and we advocate to use global limits only.
2. If we keep it, then please name it `MaxLocalExemplarsPerUser` for clarity (like we do for series), and same for YAML filed / CLI flag to keep it more clear